### PR TITLE
fix (swap)

### PR DESCRIPTION
**Pull request by SorinC6**

### CODE REVIEW

File: packages/swap/src/adapters/transformers.ts [hunk 1]
```diff
@@ -113,13 +113,16 @@ export const asYoroiAmount = (openswapAmount: {
...
-  const {amount, token} = openswapAmount
-  const [assetName = '', policyId = ''] = token.split('.')
-  const subject = `${assetName}.${policyId}`
-  return {
-    quantity: amount ? (amount as Balance.Quantity) : '0',
-    tokenId: subject.length === 1 ? '' : subject,
-  } as const
+  if (openswapAmount?.amount) {
+    const {amount, token} = openswapAmount
+    const [assetName = '', policyId = ''] = token.split('.')
```
Comment (stackchain) [inline packages/swap/src/adapters/transformers.ts]: is it in the right order? usually is `policyId.assetName` just wondering

File: packages/swap/src/adapters/transformers.ts [hunk 2]
```diff
@@ -113,13 +113,16 @@ export const asYoroiAmount = (openswapAmount: {
...
+    const subject = `${assetName}.${policyId}`
+    return {
+      quantity: amount ? (amount as Balance.Quantity) : '0',
```
Comment (stackchain) [inline packages/swap/src/adapters/transformers.ts]: there is a helper `asQuantity` that will do it.

File: packages/swap/src/adapters/transformers.ts (hunk 1 above)
Comment (SorinC6) [reply packages/swap/src/adapters/transformers.ts]: will change the order 😁

File: packages/swap/src/adapters/transformers.ts (hunk 2 above)
Comment (SorinC6) [reply packages/swap/src/adapters/transformers.ts]: will need to bring the helper into the package 🤔

**Review by jorbuedo — APPROVED** [review]

File: packages/swap/src/adapters/transformers.ts (hunk 2 above)
Comment (stackchain) [reply packages/swap/src/adapters/transformers.ts]: aaaa it's not exposed yet in the `yoroi/wallets`

**Review by stackchain — APPROVED** [review]

**Pull request merged**